camel-core/src/main/java/org/apache/camel/component/file/GenericFileProducer.java

```diff
@@ -274,7 +274,11 @@ public void writeFile(Exchange exchange, String fileName) throws GenericFileOper
             log.trace("About to write [{}] to [{}] from exchange [{}]", new Object[]{fileName, getEndpoint(), exchange});
         }
 
-        boolean success = operations.storeFile(fileName, exchange);
```

Comment:
We do NOT want changes in the core file producer. Try to make changes only to the camel-zipfile component